src/platform/plugins/shared/dashboard/server/api/dashboard_state_schemas.ts

```diff
@@ -166,6 +166,7 @@ export function getDashboardStateSchema() {
         defaultValue: [],
       })
     ),
+    projectRouting: schema.maybe(schema.string()),
```

Comment:
Can you make this snake case  `project_routing`. We are actively moving all dashboard state to snake case and no longer accepting new state in camel case.